std/conv.d

```diff
@@ -3275,7 +3275,20 @@ Target parse(Target, Source)(ref Source s)
     assert(b == true);
 }
 
-// input range to null literal conversions
```

Comment:
It seems like this function wasn't exposed to the user before. Is there a good use case where a user might need this explicit documentation?